freshmaker/handlers/koji/rebuild_images_on_parent_image_build.py

```diff
@@ -176,7 +176,7 @@ def _verify_advisory_rpms_in_container_build(self, errata_id, container_build_id
         # and use set of the nvrs as a value.
         advisory_rpms_by_name = {}
         e = Errata()
-        build_nvrs = e.get_builds(errata_id)
+        build_nvrs = e.get_binary_rpm_nvrs(errata_id)
         if build_nvrs:
```

Comment:
Since we have binary rpms from advisory already, there is no necessary to get binary rpms from brew and then compare. Also you're calling `session.get_build_rpms` with binary rpm nvr provided, which is wrong.

Reply:
Wait, so we don't need that `_verify_advisory_rpms_in_container_build` method at all or what?

Reply:
We need that, but we don't need to query brew for binary rpms (`build_rpms = session.get_build_rpms(build_nvr)`) now, since we already have the binary rpms which are fetched from errata.

Reply:
Ok, so I will replace it with this:
```
binary_rpm_nvrs = e.get_binary_rpm_nvrs(errata_id)
if binary_rpm_nvrs:
    for nvr in binary_rpm_nvrs:
         parsed_nvr = rpmlib.parse_nvr(nvr)
         if parsed_nvr['name'] not in advisory_rpms_by_name:
             advisory_rpms_by_name[parsed_nvr['name']] = set()
         advisory_rpms_by_name[parsed_nvr['name']].add(parsed_nvr['nvr'])
```